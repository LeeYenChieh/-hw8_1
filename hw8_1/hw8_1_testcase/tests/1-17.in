C2NPz9 TZaY6Gpv4pdLKtLWqQ6qUnercXWK9kSglt2zvlYuBSSNhQjWC0iX9A1px
Qugcw9lBptM4bueONtZwMW9ZZyQzX1kRrzuNpQakEciUx 5e5sWMY0yNkLbBop012JxVp7wFGqaOXCKobGHBXypBpOH2LuyjSZs
7K0y1UBE82x9jntZRiyFsuPftZBLwqBJ9QP3BveC0y cPc0ZYVzEoTWTDsv4ExcLU8RVriC2SitD7fWuaWzicvWkkpcbCK
iftuoUaku1nVe31l6D2DWDE7s4rMs6ASw6cV4gNgQr 0t1JcrE9U33LSkpxC4Arbk5ua3GzdeBmGZQxI
aMXs6LuGFXKQUNKk5Tpk1gD0 2D53An3a8FuqaeJiAF3wlvxd1iqQh2
NwtDGP3I9ld7nxQA7HczaZcw8h iCeIwSvVCq6MbfjsMm
C2NPz9 display
nZAicWEjm9PFR6Utq9iBG48MqH15O1 jQTlt
3083z0431K5t 0PO92DPMjbxpiPB50vclItgdp5Iqi6xFsoEx1er3JMDp4GwU7Yj
iftuoUaku1nVe31l6D2DWDE7s4rMs6ASw6cV4gNgQr display
nZAicWEjm9PFR6Utq9iBG48MqH15O1 hOOjyreNtuw2Z2RsqLJ0xypDASGMxHGTVzoQoYhn
WV2zAvUePyYzr4WTiD8sGnNlTk4UX0gnhByxyoc8d2 jXehodllpjCX7lyXdjG1UyPcWlsXDbXCN5Hv6Ia5c6GnlQmqCCJOhxMBAHSi
heK7ITNhX0pQmpE9tauBs8RBgA3pKvLUNGAZjcgjZLgYpg2g3AIl2cjl5D 3dmCQLUdBUcnW9cUO
aMXs6LuGFXKQUNKk5Tpk1gD0 display
Qugcw9lBptM4bueONtZwMW9ZZyQzX1kRrzuNpQakEciUx add scrollBar thickBlackBorder
