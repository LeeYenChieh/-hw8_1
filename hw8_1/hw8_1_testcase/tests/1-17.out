TZaY6Gpv4pdLKtLWqQ6qUnercXWK9kSglt2zvlYuBSSNhQjWC0iX9A1px 
0t1JcrE9U33LSkpxC4Arbk5ua3GzdeBmGZQxI 
2D53An3a8FuqaeJiAF3wlvxd1iqQh2 
